clients/vscode-hlasmplugin/README.md

```diff
@@ -53,6 +53,8 @@ To do this, set up two configuration files — `proc_grps.json` and `pgm_conf.js
 
 Follow the section [External Macro Libraries and COPY Members](#External-Macro-Libraries-and-COPY-Members) below for more detailed instructions on configuring the environment.
 
+The `pgm_conf.json` file may be provided implicitly by another product that supports integration with the HLASM Language support (e.g. Endevor Bridge for Git integration).
```

Comment:
The `pgm_conf.json` file can be provided implicitly by another product that supports integration with HLASM Language Support (e.g. Endevor Bridge for Git).